scripts/template.ejs

```diff
@@ -0,0 +1,45 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <style>
+        body {
+            width: 100vw;
+            height: 100vh;
+            margin: 0;
+        }
+        #page {
+            display: flex;
+            width: 100%;
+            height: 100%;
+        }
+    </style>
+    <title>Document</title>
+  </head>
+  <body>
+    <canvas id="page"></canvas>
+    <script src="https://unpkg.com/pdfjs-dist@2.0.489/build/pdf.min.js"></script>
```

Comment:
I wonder if it might make sense for us to install [pdfjs-dist](https://www.npmjs.com/package/pdfjs-dist) using the `package.json` and then load it relative from there using `node_modules/pdfjs-dist/build/pdf.js` instead from a fixed CDN version?

Reply:
@edsu So I think there may be two concerns hiding here:

1. Should we load `pdfjs-dist` via CDN or manage it in Yarn?
2. Should we keep `pdfjs-dist` up to date?

For the first concern, I'm not sure how to do that (to have this EJS template load a library from a local path, if I'm understanding correctly). I tried adding `pdfjs-dist` via yarn and then adding a `const pdfjsLib = require("yada yada")` but none of the `yada yada` incantations I tried worked.

For the second concern, it is true that the code as currently written uses a pretty out of date version of the library. I'd be happy to spend some time at least pinning it to the most recent version of `pdfjs-dist` and getting that to work. I'm a little concerned about managing `pdfjs-dist` as a dependency given there isn't much testing in `was_robot_suite` currently; my fear is that a weekly dependency update will bump this and we won't know it's broken until it's been in prod and someone notices. Maybe I'm overthinking it?

Reply:
Does it work if you use this in the template instead of the `require()` ?

```
<script src="../node_modules/pdfjs-dist/build/pdf.js"></script> 
```

Reply:
For concern 2 we could add a test of that `script/screenshot.js` is behaving as expected?

Reply:
@edsu Alas, that relative path incantation change produces a blank screenshot.

Reply:
I think I've got a fix for loading pdf.js locally instead of from CDN. Let me know if you want me to push it up.